### DISCUSSION TITLE
Timing points in editor should not auto-select in accordance to the playhead

### DISCUSSION THREAD

**Discussion by nagi-desuuu** [ppy/osu#23958] [Q&A] [answered]
Scenario:
You set a timing point, say, one beat too late, and want to seek back to the editor to fix it's position.

What is expected: (Timing Setup Panel is hidden for some reason)

https://github.com/ppy/osu/assets/75596775/191d47d0-787c-4389-b816-2497d4eb6743

What is actually happening:

https://github.com/ppy/osu/assets/75596775/9fbb22a3-a766-4c4b-9de8-5a71bd0ae7a1

(I'm unsure if it's intentional for the metronome display, but I don't know how many people also find this annoying, so I'm just going to put this in here.)

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/issues/23147
